admin/src/components/ContributionMessages/ContributionMessagesFormular.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="contribution-messages-formular">
+    <div>
+      <b-form @submit.prevent="onSubmit" @reset.prevent="onReset">
+        <b-form-textarea
+          id="textarea"
+          v-model="form.text"
+          placeholder="Enter something..."
+          rows="3"
+          max-rows="6"
+        ></b-form-textarea>
+        <b-row class="mt-4 mb-6">
+          <b-col>
+            <b-button type="reset" variant="danger">{{ $t('form.reset') }}</b-button>
+          </b-col>
+          <b-col class="text-right">
+            <b-button type="submit" variant="primary">{{ $t('form.submit') }}</b-button>
+          </b-col>
+        </b-row>
+      </b-form>
+    </div>
+  </div>
+</template>
+<script>
+import { adminCreateContributionMessage } from '@/graphql/adminCreateContributionMessage'
+
+export default {
+  name: 'ContributionMessagesFormular',
+  props: {
+    contributionId: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      form: {
+        text: '',
+      },
+    }
+  },
+  methods: {
+    onSubmit(event) {
```

Comment:
I get an error toasted localy:

<img width="1646" alt="Bildschirmfoto 2022-08-30 um 11 09 17" src="https://user-images.githubusercontent.com/25344101/187399847-582623da-160a-43ff-b1e2-55df43490728.png">

Reply:
is changed